### DISCUSSION TITLE
arenas

### DISCUSSION THREAD

**Discussion by L30nard0100b** [diamond-rip/Eden#122] [Q&A] [answered]
how can I copy and paste arenas from different worlds? (which is the fawe command?)
do i need to put another world in the server which contains the map that i would like to setup?

**Comment by GoodestEnglish** [comment, marked as answer]
First of all, I would recommend to put all your arenas and your lobby into a single world. This can avoid the dirt screen which happens when you teleport into another world. But this is not mandatory.

Eden allows you to copy and paste arena automatically, only if it is in the same world. If you want to copy to different world, unfortunately you have to redo the arena setup process.

**Reply by L30nard0100b** [reply]
Oh so I need a map with all worlds in it.
Do you know where can I find it?

**Reply by GoodestEnglish** [reply]
All maps should be in a single world. Not all world should be in a map.
Eden will not provide the arena build. You will have to find it by yourself.

**Reply by L30nard0100b** [reply]
So I need to build arenas by myself and I can't copy and paste it from another world

**Reply by GoodestEnglish** [reply]
If you mean the build, you can use worldedit //copy and //paste feature to do it. But it will not copy the arena data.

**Reply by L30nard0100b** [reply]
Yes, but how can I teleport tho another world bc my lobby and my arena are in different worlds for the moment.

**Reply by GoodestEnglish** [reply]
> Yes, but how can I teleport tho another world bc my lobby and my arena are in different worlds for the moment. 

Sorry for the late reply, tag me next time if i didnt response.
You will need an extra plugin like multiverse or multiworld which allows you to teleport to another world.
